### ISSUE TITLE
Breakdown lost some addressess?

### ISSUE THREAD

**Issue by Jorvanus** [LukePrior/nbn-upgrade-map#326]
Surely, there aren't just addresses that aren't there anymore?  This doesn't look quite right. 

![image](https://github.com/LukePrior/nbn-upgrade-map/assets/89010934/35291771-3702-408b-9d35-390d96034f70)

**Comment by LukePrior** [comment]
Hmmmm that is really strange, I think something changed because on all the recent commits addresses are getting deleted.

**Comment by LukePrior** [comment]
Not sure if @lyricnz has any idea but there haven't been any major changes on our end so looks like something might have changed on the NBN side?

**Comment by captbonus** [comment]
Could this be due to several of the garbled LocIDs being trimmed? I have noticed between the 13/1 and 11/2 locations that had Loactions without a LocID infront have disappeared from the map. have attached photos for reference

![image](https://github.com/LukePrior/nbn-upgrade-map/assets/161430568/31d83ad5-a629-4e61-99a9-04a37f016125)

![image](https://github.com/LukePrior/nbn-upgrade-map/assets/161430568/b971d978-8b93-4f49-b886-98d394d8efb5)

**Comment by Jorvanus** [comment]
> Could this be due to several of the garbled LocIDs being trimmed? I have noticed between the 13/1 and 11/2 locations that had Loactions without a LocID infront have disappeared from the map. have attached photos for reference
> 
> ![image](https://private-user-images.githubusercontent.com/161430568/308079593-31d83ad5-a629-4e61-99a9-04a37f016125.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDkwMjA0MDQsIm5iZiI6MTcwOTAyMDEwNCwicGF0aCI6Ii8xNjE0MzA1NjgvMzA4MDc5NTkzLTMxZDgzYWQ1LWE2MjktNGU2MS05OWE5LTA0YTM3ZjAxNjEyNS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMjI3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDIyN1QwNzQ4MjRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT00MTgwN2NhMjkzNTkwMjc1NDY0Y2U5MDRmYThhOWM0MjA2OGIwMDY2MjgwZjQyNzgwZDJmYzhjZGFiMDFlYzUzJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.Pwi5452Qg5ne6s2RpIk1sfkSdvgP_aZ8aIGHe3JL9Qc)
> 
> ![image](https://private-user-images.githubusercontent.com/161430568/308080540-b971d978-8b93-4f49-b886-98d394d8efb5.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDkwMjA0MDQsIm5iZiI6MTcwOTAyMDEwNCwicGF0aCI6Ii8xNjE0MzA1NjgvMzA4MDgwNTQwLWI5NzFkOTc4LThiOTMtNGY0OS1iODg2LTk4ZDM5NGQ4ZWZiNS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMjI3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDIyN1QwNzQ4MjRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lMGI0MTcwYzQ3YTk0NGI1OTA5NGIwZGVhNDVkZDg1YTQzM2EzZTg0ODQ3Zjc3OTVhZTIxZDEyN2Y5M2ViZWE2JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.f0KjMMHeHFsRMh7g7W_XJo85sYG6brX7exCcNb5T0aM)

Yeah, and if you plug that address into NBN, it acknowledges the address, but says more work needs to be done for this site to be connected.  

All the ones I had a look at, all looked like sensible removals to me, as they don't have any active NBN connections to them.

**Comment by LukePrior** [comment]
Looking through the records while a lot of the addresses getting removed are ones without a valid LocID, some do have valid IDs.

**Comment by LukePrior** [comment]
The script should be using the listed locID in the results file and just querying NBN for updated details.

NBN has updated their autocomplete API to be less generous with matching it seems but I'm not sure why we are seemingly redoing this search...

**Comment by LukePrior** [comment]
Ok this is interesting it's a 404 error: https://github.com/LukePrior/nbn-upgrade-map/actions/runs/7946125610/job/21693534967

**Comment by LukePrior** [comment]
I think it has to do with `extended_get_nbn_loc_id`

**Comment by Jorvanus** [comment]
Not sure how you want me to test, but link doesn't work, and that line, gives me error in powershell.  

That LOCID does work on the script I used to run myself. 
irm -Headers @{'referer'='https://www.nbnco.com.au/'} https://places.nbnco.net.au/places/v2/details/ChIJba6w8Tik12oRtX2Iuq5D51k | fl


timestamp           : 1709024991023
servingArea         : @{csaId=CSA300000010073; techType=WIRELESS; serviceType=Fixed wireless; serviceStatus=available;
                      serviceCategory=brownfields; rfsMessage=Jul 2013; description=WCA-Willowmavin}
addressDetail       : @{latitude=-37.30807800000001; longitude=144.8932779; reasonCode=FW_SA; serviceType=Fixed
                      wireless; serviceStatus=available; techType=WIRELESS; ee=False}
addressSplitDetails : @{address1=150 High Park Rd; locality=Forbes; state=VIC; postcode=3764}

**Comment by Jorvanus** [comment]
![image](https://github.com/LukePrior/nbn-upgrade-map/assets/89010934/9a6b0bed-3011-4fc7-b6b0-ed76d4e2fd2e)

**Comment by LukePrior** [comment]
I've set it to rerun, seems the new addresses have been fully removed from NBN so I guess not too much of an issue.

https://github.com/LukePrior/nbn-upgrade-map/actions/runs/8062401331/job/22022041834

**Comment by LukePrior** [comment]
Will revisit in a few weeks but hopefully it should stabilise, once it does can look into adjusting stats

**Comment by LukePrior** [comment]
Seems to be fine now.
